src/BSML/Tags/LeaderboardTag.cpp

```diff
@@ -13,14 +13,14 @@ using namespace UnityEngine;
 namespace BSML {
     static BSMLNodeParser<LeaderboardTag> leaderboardTagParser({"leaderboard", "custom-leaderboard"});
 
-    GlobalNamespace::LeaderboardTableView* leaderboardTemplate = nullptr;
     UnityEngine::GameObject* LeaderboardTag::CreateObject(UnityEngine::Transform* parent) const {
-        if (leaderboardTemplate == nullptr) {
+        static SafePtrUnity<GlobalNamespace::LeaderboardTableView> leaderboardTemplate;
```

Comment:
Please look at other files with template getters and implement this like those (ButtonTag for example)

Reply:
oh sorry I guess I didn't look closely enough at how the other tags implemented it